src/go/k8s/config/without-webhook/manager_auth_proxy_patch.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     spec:
       containers:
       - name: kube-rbac-proxy
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.1
```

Comment:
Tag doesn't exist https://console.cloud.google.com/gcr/images/kubebuilder/GLOBAL/kube-rbac-proxy

Reply:
I overlooked that the repo for kube-rbac-proxy was using quay as the source repo. It exists there https://quay.io/repository/brancz/kube-rbac-proxy?tab=tags&tag=latest

I've changed the config, so it will pull from quay instead